assets/js/util/scroll.js

```diff
@@ -65,6 +65,47 @@ export function getHeaderHeight( breakpoint ) {
 	return headerHeight;
 }
 
+/**
+ * Returns the height of the sticky WordPress admin bar, if present.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} breakpoint The current breakpoint.
+ * @return {number} The height of the sticky WordPress admin bar, if present.
+ */
+function getWordPressAdminBarHeight( breakpoint ) {
```

Comment:
I can confirm the `BREAKPOINT_SMALL` / `BREAKPOINT_TABLET` crossover does correlate precisely to the breakpoint at which the WordPress admin bar changes its stickiness.

However, it's a valid concern anyway, I think, considering that `useBreakpoint()` uses a throttled value from `useWindowWidth()`, which means it could be using a stale value when determining the header height...

I think the additional defensive measures being proposed here would be enough to mitigate this edge case, though.